isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend/spring.adoc

```diff
@@ -95,8 +88,13 @@ public class IsyFactApplication extends SpringBootServletInitializer {
 }
 ----
 
+Mit diesem Vorgehen wird explizite Kontrolle über die Konfiguration und das Komponenten-Scannen geboten.
```

Comment:
Wäre hier `@SpringBootConfiguration` nicht als besserer Standardfall zu benennen, wenn diese explizite Kontrolle benötigt wird?

Reply:
Der Unterscheid zwischen den Annotationen ist das in @SpringBootConfiguration noch die @Indexed Annotation dabei ist, die für die Erzeugung einer Index Datei gebraucht wird. Was nur hauptsächlich Vorteile bei der Startup Zeit von großen Appliaktionen bietet oder gemäß der Spring Boot Doku bei Tests hilfreich ist. (Vgl. "@SpringBootConfiguration: enable registration of extra beans in the context or the import of additional configuration classes. An alternative to Spring’s standard @Configuration that aids [configuration detection](https://docs.spring.io/spring-boot/reference/testing/spring-boot-applications.html#testing.spring-boot-applications.detecting-configuration) in your integration tests.) Aus diesem Grund, denke ich nicht, dass das Vorteile bringt, durch die Extra Index Datei und die extra Möglichkeit aber ein mehr an Komplexität was einmal zu Verwirrung bei den Anwenden führen kann und der Index File ggf. in der CI/CD Pipeline berücksicht werden müsste. Daher ist die Empfehlung, alles wie bisher zu machen wenn die Kontrolle notwendig ist oder die Standard SpringBoot Implementierung mit der @SpringBootApplication Annotation zu verwenden, sodass jeder Spring Entwickler mit Bekanntem arbeitet. Können hier aber gerne nochmal zu sprechen.